OpenOversight/app/main/views.py

```diff
@@ -288,10 +288,16 @@ def get_tutorial():
 @login_required
 def profile(username: str):
     if re.search("^[A-Za-z][A-Za-z0-9_.]*$", username):
-        user = User.by_username(username).one()
+        try:
+            user = User.by_username(username).one()
+        except NoResultFound:
+            abort(HTTPStatus.NOT_FOUND)
     else:
         abort(HTTPStatus.NOT_FOUND)
 
+    if current_user.username != user.username:
+        abort(HTTPStatus.FORBIDDEN)
```

Comment:
Make sure users can only see their own profiles.

Reply:
I think users are expected to be able to see other users' profiles since the leaderboard links to user profiles:
https://github.com/lucyparsons/OpenOversight/blob/43f3ac9dbd89cb33aeb957f0523580e733e941e5/OpenOversight/app/templates/leaderboard.html#L29

Admins can also use it to audit user classifications

Reply:
Dope. I was running off the previous ticket. I'll update the logic and tests.